README.md

```diff
@@ -187,3 +187,101 @@ After that you can start QEMU
 ```shell
 qemu-system-x86_64 -serial stdio -drive file=trenchboot.img,if=virtio -enable-kvm
 ```
+
+## Development
+
+Usually after making changes to recipes it's enough to just run [build](#build)
+again but sometimes not all changes are detected. In those cases you can clean
+changed recipe and then rebuild the project.
+
+```shell
+kas-container shell meta-trenchboot/kas-generic-tb.yml -c "bitbake -c cleansstate <recipes>"
+kas-container build meta-trenchboot/kas-generic-tb.yml
+```
+
+Replace `<recipes>` with name of changed recipes e.g. `skl`, `grub`,
+`linux-tb` and then rebuild.
+
+### Source revision
+
+To change branch or commit used by a recipe you have to change `BRANCH` or
+`SRCREV` variable in appropriate recipe file. In case of Linux kernel those
+variables are named `KBRANCH` and `SRCREV_machine`
+
+Below is list of recipe files for main TB components
+
+```text
+intel-sinit-acm - recipes-support/intel-sinit-acm/intel-sinit-acm_630744.bb
+skl - recipes-support/skl/skl_git.bb
+linux-tb - recipes-kernel/linux/linux-tb_6.6.bb
+grub & grub-efi (common parts) - recipes-bsp/grub/grub-tb-common.inc
```

Comment:
Is it just me, or rebuilding grub fails with:

```
| Updating file build-aux/config.rpath (backup in build-aux/config.rpath~)
| Updating file m4/extern-inline.m4 (backup in m4/extern-inline.m4~)
| gnulib/gnulib-tool: *** file /build/workspace/sources/grub/gnulib/m4/lib-ld.m4 not found
| gnulib/gnulib-tool: *** Stop.
| /build/workspace/sources/grub/bootstrap: gnulib-tool failed
| WARNING: exit code 1 from a shell command.
```

Reply:
Yes, after each build you have to delete `gnulib` directory (I don't know why).
I thought I added this information but either I forgot or I somehow didn't copy it.

Reply:
We need to document it as a known issue, and ideally resolve - if not now, then in the future.

Reply:
Added short description: 3b95b995b171615755042b0c6492186d893c38b2.
Added issue: https://github.com/3mdeb/meta-trenchboot/issues/31